pt-BR/routing.md

```diff
@@ -1245,42 +1245,42 @@ $ rails routes -c Comments
 $ rails routes -c Articles::CommentsController
 ```
 
-TIP: You'll find that the output from `rails routes` is much more readable if you widen your terminal window until the output lines don't wrap.
+TIP: Você irá chegar a conclusão que o resultado de `rails routes` é muito mais legível se você ampliar a janela do seu terminal até que não haja quebra de linha.
 
-### Testing Routes
+### Testando Rotas
 
-Routes should be included in your testing strategy (just like the rest of your application). Rails offers three [built-in assertions](https://api.rubyonrails.org/classes/ActionDispatch/Assertions/RoutingAssertions.html) designed to make testing routes simpler:
+Rotas deveriam ser incluidas na sua estratégia de testes (assim como resto da sua aplicação). Rails oferece três [validações nativas](https://api.rubyonrails.org/classes/ActionDispatch/Assertions/RoutingAssertions.html) desenvolvidas para fazer os testes de rotas mais simples:
```

Comment:
O que vocês acham? Pra mim ajuda a enfatizar a postura favorável a testes dentro do Rails, algo que é característica do framework.

```suggestion
Rotas devem ser incluidas na sua estratégia de testes (assim como resto da sua aplicação). Rails oferece três [validações nativas](https://api.rubyonrails.org/classes/ActionDispatch/Assertions/RoutingAssertions.html) desenvolvidas para fazer os testes de rotas mais simples:
```

Reply:
No caso de rotas, acho que depende muito da lógica que a pessoa tem no projeto dela. Se a rota for mais complexa, algo como direcionar pra um lugar X se receber um parâmetro diferente ou se a pessoa estiver autenticada no app, aí faz sentido ter teste.

Pros casos mais genéricos como esse abaixo, não vejo razão pra botar teste:
```ruby
get 'profile', to: 'users#show'
```

Agora, se for algo como:
```ruby
constraints(id: /[A-Z][A-Z][0-9]+/) do
  resources :photos
  resources :accounts
end
```

Aí vale a pena ter um teste, até como especificação do que você quer implementar, seguindo uma abordagem mais TDD. O mesmo vale pra *routers* personalizados, se o seu app precisar.

Reply:
Dito isso, pelo que eu li, o artigo não coloca essa ênfase em testes. Eu tô OK com `deveriam`. :)

Reply:
entendo com certeza e valeu !! 😄